src/cds/build.gradle.kts

```diff
@@ -6,8 +6,6 @@ plugins {
     alias(libs.plugins.kotlin.serialization)
     alias(libs.plugins.dokka)
     alias(libs.plugins.protobuf)
-    base
-    idea
```

Comment:
Idea plugin is required for better tests integration with IDEA, it's able to show diffs then. What's wrong with it?

Reply:
Can you express your idea of better integration? AFAIK, there are two directions in IDEA<->Gradle interation:
1. `idea` plugin allows to generate iml files to express unaware IDEA of project structure without relying on Gradle. However, you also can manually provide additional information to project structure.
2. IDEA with attached Gradle project uses a different approach of a special init-plugin that configures Gradle to behave as needed (including structured test output). While that does consume additional configuration from `idea` plugin, we're not using it anyways.

So in my eyes there is no need for that plugin, but I might be missing your experience and would like to learn. Did it help you? After removing it tests run good for me in IDEA.